src/components/card/index.tsx

```diff
@@ -42,11 +42,11 @@ export type CardProps = ViewProps &
     /**
      * card custom width
      */
-    width?: number | string;
+    width?: DimensionValue;
```

Comment:
`DimensionValue` doesn't support `string` value without `%`.
Do we want to remove the `string` support  from these props?

Relevant for all the components with `DimensionValue` type.

Reply:
What do you mean by "remove `string` support"? 
Do we do anything special with the strings? we just pass it on to style props..